File: Codigo/Angular/Ejemplo Homeworks y angular material/homeworks-app/src/app/models/Homework.ts
import { Exercise } from './Exercise';

export class Homework {
  id: number;
  description: string;
  dueDate: Date;
  score: number;
  //exercises: Array<Exercise>;

  constructor(description:string, score:number, dueDate:Date){//, exercises: Array<Exercise>){
      this.description = description;
      this.score = score;
      this.dueDate = dueDate;
      //this.exercises = exercises;
  }
}

File: Codigo/Clase 4/Ejemplo/Ej.Domain/Homework.cs
using System;

namespace Ej.Domain
{
    public class Homework
    {
        public int Id { get; set; }
        public string Description { get; set; }
        public DateTime DueDate { get; set; }
        public int Score { get; set; }
        public Homework() { }
    }
}
